bytes_ostream.hh

```diff
@@ -139,7 +139,7 @@ private:
     // size must not be zero.
     [[gnu::always_inline]]
     value_type* alloc(size_type size) {
-        if (__builtin_expect(size <= current_space_left(), true)) {
+        if (size <= current_space_left()) [[likely]] {
```

Comment:
The existing annotation was wrong. We can't say that there will be enough space with high probability. The patch isn't wrong though, it doesn't make it worse.